pkgs/swift2objc/lib/src/generator/generators/class_generator.dart

```diff
@@ -144,8 +144,27 @@ List<String> _generateClassProperty(PropertyDeclaration property) {
   if (property.isStatic) {
     header.write('static ');
   }
+  final prefixes = [
+    if (property.unowned) 'unowned',
+    if (property.weak) 'weak',
+  ];
+
+  var prefix = prefixes.isEmpty ? '' : '${prefixes.join(' ')} ';
+  var propSwiftType = property.type.swiftType;
 
-  header.write('public var ${property.name}: ${property.type.swiftType} {');
+  header.write('public ${prefix}var ${property.name}: $propSwiftType {');
+
+  if (property.lazy) {
```

Comment:
Is this special case necessary? Seems like the existing logic below should work fine. The code generator can basically just ignore the lazy annotation